### ISSUE TITLE
Failed to initialize QMediaPlayer "Not available"

### ISSUE THREAD

**Issue by jinwa7262** [Nuitka/Nuitka#2868]
Works fine before packaging with Nuitka, error after packaging with Nuitka: No QtMultimedia backends found. Only QMediaDevices, QAudioDevice, QSoundEffect, QAudioSink, and QAudioSource are available.
Failed to initialize QMediaPlayer "Not available", causing mp3 files to not play normally
After checking, it is because the two dll files in PySide6\plugins\multimedia\ cannot be found, but I copied the multimedia directory to PySide6\plugins\ and it still doesn't work.

- Nuitka version, full Python version, flavor, OS, etc. as output by *this exact* command.

  > python -m nuitka --version

2.2.2
Commercial: None
Python: 3.12.0 (tags/v3.12.0:0fb18b0, Oct  2 2023, 13:03:39) [MSC v.1935 64 bit (AMD64)]
Flavor: CPython Official
Executable: I:\Python\PythonProject\speechVoice\.venv\Scripts\python.exe
OS: Windows
Arch: x86_64
WindowsRelease: 11
Version C compiler: d:\Program Files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.38.33130\bin\Hostx64\x64\cl.exe (cl 14.3).

- How did you install Nuitka and Python

> pip install nuitka

installed in virtualenv

- The specific PyPI names and versions

  It should be taken from this output if there specific packages involved, e.g. `numpy`, you are
  expected to shorten this to the relevant ones.

  > python -m pip freeze
aiohttp==3.9.5
aiosignal==1.3.1
attrs==23.2.0
certifi==2024.2.2
edge-tts==6.1.11
frozenlist==1.4.1
idna==3.7
imageio==2.34.1
multidict==6.0.5
Nuitka==2.2.2
numpy==1.26.4
ordered-set==4.1.0
pillow==10.3.0
PySide6==6.7.0
PySide6_Addons==6.7.0
PySide6_Essentials==6.7.0
shiboken6==6.7.0
yarl==1.9.4
zstandard==0.22.0

I tried to build the project like this:
> Nuitka --standalone --enable-plugin=pyside6 mainForm.py

Nuitka log:

>Nuitka-Options: Used command line options: --standalone --enable-plugin=pyside6 mainForm.py
Nuitka:WARNING: The Python version '3.12' is only experimentally supported by Nuitka '2.2.2', but an upcoming release will change that. In the mean time use Python version '3.11' instead or newer Nuitka.
Nuitka: Starting Python compilation with Nuitka '2.2.2' on Python '3.12' commercial grade 'not installed'.
Nuitka-Plugins:options-nanny: Note, when using 'PySide6', consider using '--disable-console' option. Otherwise a terminal window will open. However for debugging, terminal output is the easiest way to see
Nuitka-Plugins:options-nanny: informative traceback and error information, so delay this until your program is working and remove once you find it non-working, and use '--enable-console' to make it explicit
Nuitka-Plugins:options-nanny: and not see this message.
Nuitka: Completed Python level compilation and optimization.      
Nuitka: Generating source code for C backend compiler.
Nuitka: Running data composer tool for optimal constant value handling.      
Nuitka: Running C compilation via Scons.
Nuitka-Scons: Backend C compiler: cl (cl 14.3).
Nuitka-Scons: Backend linking program with 104 files (no progress information available for this stage).
Nuitka-Scons: Compiled 104 C files using clcache with 104 cache hits and 0 cache misses.
Nuitka-Plugins:pyside6: Including Qt plugins 'iconengines,imageformats,platforms,styles,tls' below 'PySide6\qt-plugins'.
Nuitka-Plugins:data-files: Included data file 'certifi\cacert.pem' due to package data for 'certifi'.         
Nuitka: Keeping build directory 'mainForm.build'.
Nuitka: Successfully created 'mainForm.dist\mainForm.exe'.

**Comment by kayhayen** [comment]
There is no reproducer, so this is nothing to work on. For PySide6.7 I made the 2.2.3 hotfix, but that was mostly for macOS, it should be working fine. You probably are not checking the help output of the pyside6 plugin, for the likely solution though, which will be to include more Qt plugins than the default.

**Comment by WANGRUI-ZB** [comment]
I had the same problem. How did I solve it? @jinwa7262
